packages/cli/src/tools/assertRequiredOptions.ts

```diff
@@ -1,38 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * @format
- */
-import {Option} from 'commander';
-import {camelCase} from 'lodash';
-import {CommandOption} from '@react-native-community/cli-types';
-
-type PassedOptions = {
-  [key: string]: unknown;
-};
-
-// Commander.js has a 2 years old open issue to support <...> syntax
-// for options. Until that gets merged, we run the checks manually
-// https://github.com/tj/commander.js/issues/230
-export default function assertRequiredOptions(
-  options: CommandOption<any>[],
-  passedOptions: PassedOptions,
-) {
-  options.forEach((opt) => {
-    const option = new Option(opt.name);
-
-    if (!option.required) {
```

Comment:
Removing this is likely a regression. We support adding a `required` key to the command's interface, and when it's set to "true" it shall throw. Can you make sure this is still supported? Also, the linked issue is closed with a comment that it was fixed in commanderjs v4. We use v2.

Reply:
Bumped commander.js to `v4`. It should be better now.